core/domain/stats_domain.py

```diff
@@ -216,13 +330,15 @@ def validate(self):
                     self.exp_version))
 
         exploration_stats_dict = self.to_dict()
-
+        # Ignore[misc] is used here because mypy expects TypedDict key to be s
+        # string literal, but because the key is being accessed by a variable
+        # instead of a string, mypy does not recognize that the key is a string.
         for stat_property in exploration_stats_properties:
-            if not isinstance(exploration_stats_dict[stat_property], int):
+            if not isinstance(exploration_stats_dict[stat_property], int): # type: ignore[misc]
```

Comment:
Can you set the type of `exploration_stats_properties` to `Literal` (or list of literals I'm not sure rn) so that this ignore is not needed?

Reply:
Done